### DISCUSSION TITLE
Log warnings

### DISCUSSION THREAD

**Discussion by an3snortje** [rainepretorius/olarm-ha-integration#71] [Q&A]
Been getting the below warnings, they occur multiple times, seemingly continuously. Been on a drive to clear up warnings and errors from my HA, so would be appreciative if anyone has any ideas.

> This error originated from a custom integration.
> 
> Logger: custom_components.olarm_sensors
> Source: custom_components/olarm_sensors/olarm_api.py:300
> Integration: Olarm Sensors (documentation, issues)
> First occurred: 2:43:39 PM (127 occurrences)
> Last logged: 2:54:19 PM
> 
> This device's area names have not been set up in Olarm, generating automatically
> 
>

**Comment by rainepretorius** [comment]
Please name the area in your Olarm App.
